File: php/lumen/routes/web.php
<?php

/**
 * Lumen Framework Routes
 * 
 * Benchmark server routes following PHP best practices.
 */

$router->get('/', [
    'uses' => 'ApplicationController@index',
]);

$router->get('/health', [
    'uses' => 'ApplicationController@healthCheck',
]);

$router->get('/user/{id}', [
    'uses' => 'UserController@show',
]);

$router->post('/user', [
    'uses' => 'UserController@create',
]);

File: python/masonite/routes/web.py
from masonite.routes import Route


ROUTES = [
    Route.get("/", "UserController@index"),
    Route.get("/user/@id:int", "UserController@show"),
    Route.post("/user", "UserController@create"),
    Route.get("/health", "HealthController@index"),
    Route.get("/error", "ErrorController@index"),
]
